tests/resolvers/Query/users.spec.ts

```diff
@@ -80,34 +76,24 @@ describe("resolvers -> Query -> users", () => {
       };
     });
 
-    const { requestContext } = await import("../../../src/libraries");
+    const testUser = await createTestUser();
 
-    const spy = vi
-      .spyOn(requestContext, "translate")
-      .mockImplementationOnce((message) => `Translated ${message}`);
+    const args: QueryUsersArgs = {
+      orderBy: null,
+      where: {
+        id: testObjectId as unknown as string,
```

Comment:
Idk who wrote these `as unknown as string` annotations. Remove them and convert the objectIds to string before passing them into resolvers, whether in arguments, or context.